server/manager/src/main/java/org/apache/accumulo/manager/tableOps/split/UpdateTablets.java

```diff
@@ -177,7 +177,8 @@ private void addNewTablets(long tid, Manager manager, TabletMetadata tabletMetad
           continue;
         }
 
-        var mutator = tabletsMutator.mutateTablet(newExtent).requireAbsentTablet();
+        var mutator =
+            tabletsMutator.mutateTablet(newExtent, newExtent.prevEndRow()).requireAbsentTablet();
```

Comment:
I think this prevEndRow doesn't exist yet (it's set on line 187 below). SplitIT is failing, I think because of this.

Reply:
I'm wondering if maybe we should add `ConditionalTabletsMutator.createTablet(KeyExtent)` for this case.

Reply:
> I'm wondering if maybe we should add ConditionalTabletsMutator.createTablet(KeyExtent) for this case.

Yeah could do something like that, it would need to be a new top level ample method.

Reply:
Resolved this with the change in 8098890. SplitIT is passing again.